spec/shr_lab_equivalent_units_vs.txt

```diff
@@ -0,0 +1,62 @@
+Grammar:	ValueSet 5.0
+Namespace:	shr.lab
+
+CodeSystem:     UCUM = http://unitsofmeasure.org
+
+//------------- Unit of Measure Synonyms ------------
+
+ValueSet:       EquivalentUnitsVS-g-per-L
+Description:    "Units equivalent to g/L"
+UCUM#g/L        "Grams per liter"
+UCUM#mg/mL      "Milligrams per milliliter"
+UCUM#ug/uL      "Micrograms per microliter"
+
+ValueSet:       EquivalentUnitsVS-ng-per-mL
+Description:    "Units equivalent to ng/mL"
+UCUM#ng/mL      "Nanograms per milliliter"
+UCUM#ug/L       "Micrograms per liter"
+
+ValueSet:       EquivalentUnitsVS-umol-per-L
+Description:    "Units equivalent to nmol/mL"
+UCUM#nmol/mL    "Nanomoles per milliliter"
+UCUM#umol/L     "Micromoles per liter"
+
+ValueSet:       EquivalentUnitsVS-thousand-per-uL
+Description:    "Units equivalent to 10*3/uL"
+UCUM#10*3/uL    "Thousand per microliter"
+UCUM#10*6/mL    "Million per milliliter"
+UCUM#10*9/L     "Billion per liter"
+UCUM#10*3/mm3   "Thousand per cubic millimeter"
+
+ValueSet:       EquivalentUnitsVS-million-per-uL 
+Description:    "Units equivalent to 10*6/uL"
+UCUM#10*6/uL    "Million per microliter"
+UCUM#10*12/L    "Trillion per liter"
+UCUM#10*6/mm3   "Million per cubic millimeter"
+
+ValueSet:       EquivalentUnitsVS-volume-fraction
+Description:    "Units equivalently representing volume fraction"
+UCUM#%          "Percent"
```

Comment:
I would omit `Percent` in a value set which contains explicit ratios. Logically, it makes sense, however I've seen UoM conversion algorithms which treat ratios separately.